integration_test/third_party_apps_data/applications/mssql/enable

```diff
@@ -1 +1,20 @@
-'Noop; MSSQL monitoring is on by default.'
\ No newline at end of file
+# This config gets merged with the built-in Ops Agent config, which already
+# includes the v1 receiver in the default pipeline.
+# Because the v2 receiver here uses a unique ID, it does not overwrite
+# the default receiver and so both receivers will be active simultaneously
+# during the test.
+Add-Content 'C:\Program Files\Google\Cloud Operations\Ops Agent\config\config.yaml' "
+metrics:
+  receivers:
+    mssql_v2:
+      type: mssql
+      receiver_version: 2
+  service:
+    pipelines:
+      mssql_v2:
+        receivers:
+        - mssql_v2
+"
+
+Stop-Service google-cloud-ops-agent -Force
```

Comment:
we use `Restart-Service google-cloud-ops-agent -Force` over here:

https://github.com/GoogleCloudPlatform/ops-agent/blob/53018f985a2120b7b5b31110662979d4a3bc16c6/integration_test/ops_agent_test.go#L75

Seems like a slightly better way to do this than what you have, because a) it combines the stop and the start into a single command, and b) it makes me worry less about what the `*` in `Start-Service google-cloud-ops-agent*` is doing.

Reply:
@martijnvans I made this change.

Reply:
Looks like this has been addressed. Resolving this thread.